### PR TITLE
Add note tests

### PR DESCRIPTION
**Pull request by arcadiafalcone**
## Why was this change made?
Add mappings for notes that become abstracts

## How was this change tested?
n/a

## Which documentation and/or configurations were updated?
n/a

### CODE REVIEW

File: app/services/cocina/from_fedora/descriptive/notes.rb [hunk 1]
```diff
@@ -70,7 +70,7 @@ def abstract_type(node, parallel: false)
 
         def note_type(node)
           if node['type'].present? && node['type'] != 'summary'
-            { type: node['type'] }
+            { type: node['type'].downcase }
```
Comment (peetucket) [inline app/services/cocina/from_fedora/descriptive/notes.rb:73]: I actually had this downcased in my original PR, but removed it after running validations against 10k random druids and finding new roundtripping errors for note nodes.  It is not currently downcased in the main branch.

 I then added tests to ensure the case is preserved: https://github.com/sul-dlss/dor-services-app/pull/2836/files#diff-ea4ff0bb6abd64bf28b19eaa84a65730ea4ee136b92a758f7f42f3e3c1e852b6R47-R94

**Comment by peetucket** [comment]
I think the general problem I was having was that even though the unskipped tests were passing, when I ran the actual validation script, the error rate increased.  Are these new tests coming from the failed druids.

**Comment by arcadiafalcone** [comment]
Yes, they're based on the errors that I saw in looking through the validation test results -- previously the tests didn't represent all cases where notes correctly turn into abstracts when round-tripping back to MODS,

**Comment by peetucket** [comment]
> Yes, they're based on the errors that I saw in looking through the validation test results -- previously the tests didn't represent all cases where notes correctly turn into abstracts when round-tripping back to MODS,

Ok, that's cool.  Based on what I saw before, I'm thinking the downcasing of the note type in this PR may cause the roundtripping errors to come back again though.

**Comment by arcadiafalcone** [comment]
I'll remove it - I didn't think it should be there based on my understanding of the code, but only made the edit because I was trying to resolve conflicts.

**Comment by peetucket** [comment]
I'm going to try and unskip the new tests on the this branch and push commits here before merging to the other branch

**Review by peetucket — APPROVED** [review]

**Pull request merged**